Add memberships (#30)
paragraph-xyz/docs: src/pages/docs/memberships.md

---
title: Memberships
description: Monetize your Paragraph newsletter with recurring memberships.
---

> Create memberships to offer benefits to your subscribers and optionally begin monetizing with recurring payments.

Reviewer: Just an idea, not sure if it's better necessarily.

```suggestion
Create free or paid memberships to offer benefits to your subscribers and begin monetizing with recurring payments.
```

---

# Memberships on Paragraph

Paragraph enables you to create _memberships_ for your newsletter. Memberships allow you to grant additional benefits and perks to your subscribers.

> All Paragraph publications have a single free membership by default, but you can optionally create paid memberships. Upon subscribing to your newsletter, readers will be auto-join the free membership but can choose to upgrade to a paid tier (if you have any paid tiers). At any point, a subscriber can also click "Upgrade to Paid" on your publication and purchase a paid subscription.

Reviewer: ```suggestion
All Paragraph publications have a single free membership by default, but you can optionally create paid memberships. Upon subscribing to your newsletter, readers will auto-join the free membership but can choose to upgrade to a paid tier (if you have any paid tiers). At any point, a subscriber can also click "Upgrade to Paid" on your publication and purchase a paid subscription.
```

> For each membership, you can upload a mmebership image and choose additional benefits and perks that are associated to each. For example, you can offer a private Discord community to a paid membership tier. You can also gate exclusive content on Paragraph to certain memberships.

Reviewer: ```suggestion
For each membership, you can upload a membership image and choose additional benefits and perks that are associated to each. For example, you can offer a private Discord community to a paid membership tier. You can also gate exclusive content on Paragraph to certain memberships.
```

All publications also have a _membership_ page, accessible at /memberships (for example: https://paragraph.xyz/@blog/memberships). You can share this link with your readers to incentivize them to upgrade and illustrate the perks they'd get.

![Paragraph memberships](/img/membership.png)

## Paid memberships

In order to begin accepting payment for your memberships, you need to connect your Paragraph account with Stripe. You can do so in your [publication settings](https://paragraph.xyz/settings/publication/memberships-token-gating).

After conneting with Stripe you'll be able to create paid memberships and charge a monthly or yearly fee to join.

## Future improvements

This is the first iteration of what's to come. Within the next few months, memberships will be:

* Backed by NFTs. Members can mint an NFT indicating that they're a part of this membership.
* Purchasable using fiat or crypto.

> * Recurring payments, or one-off.

Reviewer: ```suggestion
* One-off payments will be possible, not just recurring.
```

---
title: Memberships
description: Monetize your Paragraph newsletter with recurring memberships.
---

Create free or paid memberships to offer benefits to your subscribers and begin monetizing with recurring payments.

---

# Memberships on Paragraph

> 

Reviewer 2: I think we should mention that you can mark one as the most popular.

For each membership, you can upload a membership image and choose additional benefits and perks that are associated to each. For example, you can offer a private Discord community to a paid membership tier. You can also gate exclusive content on Paragraph to certain memberships.

All publications also have a _membership_ page, accessible at /memberships (for example: https://paragraph.xyz/@blog/memberships). You can share this link with your readers to incentivize them to upgrade and illustrate the perks they'd get.

![Paragraph memberships](/img/membership.png)

## Paid memberships

In order to begin accepting payment for your memberships, you need to connect your Paragraph account with Stripe. You can do so in your [publication settings](https://paragraph.xyz/settings/publication/memberships-token-gating).

> After conneting with Stripe you'll be able to create paid memberships and charge a monthly or yearly fee to join.

Reviewer 2: typo: connecting

## Future improvements

This is the first iteration of what's to come. Within the next few months, memberships will be:

* Backed by NFTs. Members can mint an NFT indicating that they're a part of this membership.
* Purchasable using fiat or crypto.
* One-off payments will be possible, not just recurring.
* Memberships can be in limited quantity.

> # Importing paid subscribers

Reviewer 2: Spacing above this header is too small
